.github/CODEOWNERS

```diff
@@ -9,6 +9,10 @@
 /website/src/pages/dbt-cloud/                          @runleonarun 
 /website/static/img/docs/dbt-cloud/                    @runleonarun 
 
+# Blog content assigned to Developer experience team
+/website/blog/                       @ganz @gwenwindflower @joellabes
```

Comment:
@jasnonaz We should really use @dbt-labs/devex but the team needs to be updated to the latest members https://github.com/orgs/dbt-labs/teams/devex

Reply:
Have asked in tech support https://dbt-labs.slack.com/archives/C01AAEJU28J/p1645583663897359